scripts/buildColorTypes.js

```diff
@@ -0,0 +1,80 @@
+import * as colors from '@material-ui/core/colors';
+import * as fse from 'fs-extra';
+import * as path from 'path';
+
+// use netlify deploy preview if you want to test changes
+const HOST = 'https://material-ui.com/';
+
+function getColorHref(name, variant) {
+  return `static/colors-preview/${name}-${variant}-24x24.svg`;
```

Comment:
What's the use case for including the size of the svg?

```suggestion
  return `static/colors-preview/${name}-${variant}.svg`;
```

Reply:
Just so that we know it. Adds an easy "button" if we want to change the size and invalidate downstream caches.